docs/sdk/examples/exporttopdf.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license.
+-->
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="description" content="Export to PDF">
+	<meta name="keywords" content="ckeditor, editor, wysiwyg, ckeditor4, ckeditor 4, ckeditor, fckeditor, pdf, export, export to pdf, export to file, file, editor, wysiwyg, html, richtext, text, javascript">
+	<meta name="sdk-samples" content="Exporting content to PDF">
+	<meta name="sdk-category" content="sdk-working-with-document">
+	<meta name="sdk-order" content="15">
+	<title>Export to PDF</title>
+	<link href="https://fonts.googleapis.com/css?family=Maven+Pro:700,500,400" rel="stylesheet">
+	<link href="../template/theme/css/sdk.css" rel="stylesheet">
+	<script src="../../ckeditor4/ckeditor.js"></script>
+	<script src="assets/picoModal-2.0.1.min.js"></script>
+	<script src="assets/contentloaded.js"></script>
+	<script src="assets/simplesample.js"></script>
+	<script src="assets/beautify-html.js"></script>
+	<script src="assets/sample.js"></script>
+	<!--[if lt IE 9]>
+	<script src="assets/html5shiv.min.js"></script>
+	<![endif]-->
+	<link rel="icon" href="../template/theme/img/favicon.ico">
+</head>
+<body>
+<header class="sdk-header">
+</header>
+<section class="sdk-container">
+	<nav class="sdk-sidebar">
+	</nav>
+	<section class="sdk-contents">
+		<style>
+			#browserSupportInfo, #browserSupportInfo .ie {
+				display: none;
+			}
+		</style>
+
+		<h1>Exporting content to a PDF<a class="documentation" href="https://ckeditor.com/docs/ckeditor4/latest/features/exporttopdf.html">Documentation</a></h1>
+
+		<p>
+			CKEditor 4 operates mostly on rich text and HTML markup, which already gives tons of possible uses. And even if you need the content it created outside of the web environment, you are still able to save it to a file and use it elsewhere. This is possible thanks to the optional
+			<a href="https://ckeditor.com/cke4/addon/exportpdf">Export to PDF</a> plugin which allows saving the editor content to a PDF.
+		</p>
+		<p>
+			It introduces a <span class="button_icon" data-icon="exportpdf" title="Export to PDF">&nbsp;</span><b>new toolbar button</b>.
+			When it's clicked, data from editor is sent to the service maintained by <a href="https://ckeditor.com/ckeditor-cloud-services/">CKEditor Cloud Services</a>.
+			The editor takes care of the received response and saves it as a file.
```

Comment:
```suggestion
			When it's clicked, data from the editor is sent to the PDF service maintained by <a href="https://ckeditor.com/ckeditor-cloud-services/">CKEditor Cloud Services</a>.
			The PDF service transforms received HTML into a PDF file which is sent back to editor and saved as a PDF file.
```

WDYT?

Reply:
Actually service transforms the HTML to a binary file :slightly\_smiling\_face: besides from that - :+1:

Reply:
Well, basically every file is a binary file (and [text files are usually treated as separate category](https://www.nayuki.io/page/what-are-binary-and-text-files)) :wink: So it's both binary and a PDF file.

Reply:
But I'm fine with current wording ofc:+1: